scripts/quickstart.py

```diff
@@ -24,9 +26,20 @@
     storage_options={"anon": True},
 ).persist()
 
+# perform groupby aggregation
 result = ddf.groupby("passenger_count").tip_amount.mean().compute()
-
 print(result)
 
+# write result to S3
+result = dd.from_pandas(pd.DataFrame(result), npartitions=1)
+bucket_path = "s3://coiled-datasets/github-actions/quickstart/"
+try: 
```

Comment:
This will fail unless you pass the credentials to log in to AWS, this require a bit more work, as we need to add them as secrets and do a bit more work to get it to work. 

The other part that concerns me on running this on a cron, is we will be generating data every day for no reason. We could do something to delete it, or create a bucket specific for that to show the workflow, and clean it every day.

Reply:
As you can see the write to S3 is failing no now see https://github.com/coiled/github-actions-with-coiled/runs/6985643378?check_suite_focus=true#step:4:10

If this is a workflow you want to show, then we need to do some modifications to the action, plus provide credentials to be able to write to s3. 

I don't feel very comfortable putting your or my credentials to write to coiled-datasets. We can do something very similar to what we did over the runtime where we created an IAM user with a policy that allows just read and write, and it's restricted to one bucket that it's only purpose is get the data that will be created here. I'm also not sure I would have this bucket public. 

This will take me some time to set up I think probably 1h-ish. @rrpelgrim Would you like to be involved in the process for learning purposes or should I go ahead and do it?

Reply:
Thanks for looking into this @ncclementi. I'd like to be involved in the setup if possible. Feel free to grab some time on my calendar.